Center accordion on larger screens

joshuahenson opened the pull request:
Here is my solution. Let me know or edit it if you feel different sizes for the column would work better.

closes #6365

joshuahenson committed:
Center accordion on larger screens

SaintPeter commented:
This does a pretty decent job of centering up the map, but it catches the buttons in the process.

The go from comically narrow:
![image](https://cloud.githubusercontent.com/assets/553494/12504984/9aff76b4-c097-11e5-9a9a-2aa1fa406c46.png)

To massively wide:
![image](https://cloud.githubusercontent.com/assets/553494/12504985/a017c340-c097-11e5-90fd-406d8e1ec669.png)

I don't think the button header should be included in the change, just the body of the list.

joshuahenson committed:
Update button spacing

joshuahenson commented:
The button sizing was actually done previously by someone else. I hadn't touched them other than to shift the tab spacing in the jade file. 
I removed one of the nested rows on the buttons but while comparing it to the live site I just now noticed that the previous change also added that second button which does nothing. Should  #6358 be rolled back?

SaintPeter commented:
@joshuahenson The first button should hide completed tasks.

@QuincyLarson Can you take a look at this one.  Is this how the button sizing is supposed to work?  Is the centering action ok?

joshuahenson commented:
Ah yes, maybe I shouldn't try communicating immediately upon waking up.

Before I got distracted, I was wondering if I should have removed 
`.col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3` 
instead of 
`.col-xs-10.col-xs-offset-1.col-sm-6.col-sm-offset-3.col-md-4.col-md-offset-4`

SaintPeter commented on server/views/map/show.jade:
```diff
@@ -1,14 +1,14 @@
+          .row
+            .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+              .btn.btn-sm.btn-block.btn-primary.active#manipAll Show incomplete challenges
+              .btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all
```
I don't think you should change the button spacing at all.  Or, rather, the buttons are inside a row inside a row and they look "ok" on the current site, I think.  Can you just make the change you made below to the row that contains the accordian?  Or are they nested in some way?

joshuahenson committed:
Modify button size

joshuahenson commented:
I returned the buttons back to the size they are on the live site as they were before the filter button was added.

bugron commented:
This is on map-aside
![](http://i.imgur.com/nZ2yoPs.png)
